### PR TITLE
Add Navigation, Form for new vaccination and Vaccination details

### PR DESCRIPTION
**Pull request by joshualang**

### CODE REVIEW

File: src/common/styles/colors.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export default {
   balck: "#212121",
   grey: "#939393",
+  greySemi: "#cccccc",
   white: "#ffffff"
 }
```
Comment (dvtdyn) [inline src/common/styles/colors.js]: Könntest überlegen, ob du die Variablen in GlobalStyle unter :root einbaust

Comment (joshualang) [reply src/common/styles/colors.js]: Tried this first but it didn't work. even with global sass variables.

File: src/VaccinationCard.js [hunk 2]
```diff
@@ -4,37 +4,41 @@ import TimeAgo from "react-timeago"
 import germanStrings from "react-timeago/lib/language-strings/de"
 import buildFormatter from "react-timeago/lib/formatters/buildFormatter"
 
-import Title from "./common/Text/Title"
-import DetailsText from "./common/Text/DetailsText"
-import colors from "./colors"
+import colors from "./common/styles/colors"
+import Title from "./common/text/Title"
+import DetailsText from "./common/text/DetailsText"
+import { Link } from "@reach/router"
```
Comment (dvtdyn) [inline src/VaccinationCard.js]: Prüfen, ob hier das folgende hin muss:
import { Link } from "react-router-dom"

Comment (joshualang) [reply src/VaccinationCard.js]: In line 16 it is used to link to the details of that specific vaccination

**Pull request merged**